src/pages/Models/Details/_model-details.scss

```diff
@@ -209,9 +209,11 @@
     }
   }
 
+  // .model-details__offers, // DISABLED PANEL
+  // .model-details__remote-apps, // DISABLED PANEL
```

Comment:
Do these really need to be commented out if the panels are disabled?

Reply:
Yes, otherwise the hover state still exists but clicking does nothing.